qiskit/visualization/qcstyle.py

```diff
@@ -301,12 +301,10 @@ def set_style(current_style, new_style):
     current_style['cline'] = new_style.pop('creglinestyle', current_style['cline'])
     dtex = new_style.pop('displaytext', current_style['disptex'])
     for tex in dtex.keys():
-        if tex in current_style['disptex'].keys():
```

Comment:
Just an aside (since I know we're deleting this line) in general you should just use `tex in current_style['disptex']` and avoid creating the extra iterator view unless you actually need it.